esp/src/src/nls/hpcc.ts

```diff
@@ -1131,6 +1131,7 @@ export = {
         WUID: "WUID",
         Wuidcannotbeempty: "Wuid Cannot Be Empty.",
         WUSnapshot: "WU Snapshot",
+        WUSnapShot: "WU Snapshot"
```

Comment:
Should you also update the other language variants?

Reply:
@kunalaswani

Reply:
Checking those files now to see if they need the change as well.

Reply:
@kunalaswani - did you work out which version is preferred?

Reply:
@ghalliday all the other files were already synced in their merges, only this one required this update. @GordonSmith both versions are being used, and the change to the pervious would require us to change 8 additional older files, it was discussed that we should retain the previous and this current version.

Reply:
@ghalliday corrected, apologies on the miss in the ES file